pkg/controller/accountclaim/reuse.go

```diff
@@ -238,7 +243,7 @@ func (r *ReconcileAccountClaim) cleanUpAwsAccountSnapshots(reqLogger logr.Logger
 
 		_, err = awsClient.DeleteSnapshot(&deleteSnapshotInput)
 		if err != nil {
-			delError := fmt.Sprintf("Failed deleting EBS snapshot: %s", *snapshot.SnapshotId)
+			delError := errors.Wrap(err, fmt.Sprintf("Failed deleting EBS snapshot: %s", *snapshot.SnapshotId)).Error()
```

Comment:
Decided to wrap these in order to gain visibility into what was coming from the AWS side of things